python/tests/test_stats.py

```diff
@@ -1584,3 +1584,1066 @@ def verify(self, ts, reference_sets, focal=None):
                 mean = np.sum(T * L, axis=1) / np.sum(L)
                 assert np.allclose(mean, A[j])
         return A
+
```

Comment:
These tests are pretty self-contained, so I think it would be good to keep them in their own file. Say, ``test_coaltime_distribution.py``?